src/app/app.component.ts

```diff
@@ -7,4 +13,42 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'ng-freshcard-bootstrap-theme';
+
+  public isMobileMenuShown: boolean = false;
```

Comment:
let's avoid properties like this. They are mutable and mess up change detection process. Please always use subject instead